requirements.txt

```diff
@@ -0,0 +1,3 @@
+Django==1.8.2
+-e git+https://github.com/coolharsh55/python-opensubtitles@f37074796ee72c7efe1aa3ec2d534b4299b8fd88#egg=python_opensubtitles-master
```

Comment:
I'm proposing that the changes be made so that __python-opensubtitles__ is installable via `pip` directly -> `pip install python-opensubtitles` which also correctly gets listed in `pip freeze` without any `svn location` errors.